incubator/binding-mqtt.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/streams/application/session.will.message.disconnect.with.will.message/client.rpt

```diff
@@ -19,16 +19,19 @@ connect "zilla://streams/app0"
          option zilla:transmission "duplex"
 
 write zilla:begin.ext ${mqtt:beginEx()
-                             .typeId(zilla:id("mqtt"))
-                             .session()
-                                .clientId("one")
-                                .expiry(10)
-                                .willTopic("wills/one")
-                                .willFlags("QOS0")
-                                .willFormat("TEXT")
-                                .willPayload("client one session expired")
-                                .build()
-                             .build()}
+                           .typeId(zilla:id("mqtt"))
+                           .session()
+                              .clientId("one")
+                              .expiry(10)
+                              .willMessage(
+                                  mqtt:willMessage()
+                                      .topic("wills/one")
+                                      .flags("QOS0")
+                                      .format("TEXT")
+                                      .payload("client one session expired")
+                                      .build())
+                              .build()
+                           .build()}
```

Comment:
This is not quite right, instead we want:
```
write zilla:begin.ext ${mqtt:beginEx()
                           .typeId(zilla:id("mqtt"))
                           .session()
                              .clientId("one")
                              .expiry(10)
                              .willMessage()
                                  .topic("wills/one")
                                  .flags("QOS0")
                                  .format("TEXT")
                                  .payload("client one session expired")
                                  .build())
                              .build()
                           .build()}
```
This will require extending `MqttWillMessageBuilder` in `MqttFunctions`, and overriding `.build()` method to set the will message and return the session builder.

Reply:
Also consider calling the method `.will()` instead of `.willMessage()`.